### DISCUSSION TITLE
v2 - v3 migration

### DISCUSSION THREAD

**Discussion by bennypowers** [lukas-reineke/indent-blankline.nvim#699] [Q&A] [answered]
I installed v3 but get lots of configuration errors. It's not clear to me how the keys from v2 map to config for v3. Please consider writing a migration guide, or publishing in the README/issues/here one prepared by the community.

Thank you for publishing this helpful package.

**Comment by lukas-reineke** [comment, marked as answer]
There is a [migration guide in the wiki](https://github.com/lukas-reineke/indent-blankline.nvim/wiki/Migrate-to-version-3)

Feel free to open an issue if you have a specific question
